### ISSUE TITLE
Feature request: delete all (truncate) documents in a collection

### ISSUE THREAD

**Issue by revmischa** [typesense/typesense#1016]
It would be nice to be able to truncate all documents in a collection.

https://typesense-community.slack.com/archives/C01P749MET0/p1684198912218119?thread_ts=1684198198.800489&cid=C01P749MET0

**Comment by cmutel** [comment]
This is [now possible](https://typesense.org/docs/28.0/api/collections.html#truncate-a-collection)? I think this can be closed.

**Comment by kishorenc** [comment]
Yes, this was shipped with `v28`: https://typesense.org/docs/28.0/api/collections.html#truncate-a-collection
